python-checks/src/test/resources/checks/cdk/iamPolicyPublicAccessCheck.py

```diff
@@ -5,7 +5,7 @@
 iam.PolicyStatement(
   sid="AllowAnyPrincipal",
   effect=iam.Effect.ALLOW, # {{Access is set to "ALLOW" here.}}
-        #^^^^^^^^^^^^^^^^>
+# ^^^^^^^^^^^^^^^^^^^^^^^>
```

Comment:
Is it expected that refactoring change this behavior?

Reply:
This is expected. Normally, we highlight a complete assignment, or in this case a named argument. Eg. highlighting `true` may not be enough. However, in cases we have lists or directories, highlighting the complete argument with name could be too much. Unfortunately, we are not consistent here and we have to revisit the rules from time to time. 
As a resume from this comment, I will create a standard for highlighting which we can discuss within the bubble.